File: DSA/Prarthana_Baisoya_590024838/Day_2/q2.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter size of array: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter array elements: ");
    for(int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int largest = arr[0];
    for(int i = 1; i < n; i++) {
        if(arr[i] > largest) {
            largest = arr[i];
        }
    }

    printf("Largest element is: %d", largest);

    return 0;
}

File: DSA/NeeravKumarSharma_590018263/Day_2/Q2.java
import java.util.Scanner;

public class Q2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.err.println("Enter the size of array:");
        if (scanner.hasNextInt()) {
            int n = scanner.nextInt();
            

            if (n > 0) {
                System.out.println("Enter the elements of the array:");
                int maxVal = scanner.nextInt();
                
                for (int i = 1; i < n; i++) {
                    int currentNum = scanner.nextInt();
                    if (currentNum > maxVal) {
                        maxVal = currentNum;
                    }
                }
                
                System.out.println("Max element is:");
                System.out.println(maxVal);
            }
        }
        
        scanner.close();
    }
}